Tests/Adapter/PHPCR/PHPCRAdapterTest.php

```diff
@@ -5,17 +5,16 @@
 use PHPUnit\Framework\TestCase;
 use Vich\UploaderBundle\Adapter\PHPCR\PHPCRAdapter;
 use Vich\UploaderBundle\Tests\DummyEntity;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
```

Comment:
Can we commit `.styleci.yml` https://styleci.readme.io/docs/configuration file with required config?